cves/CVE-2003-0020.yml

```diff
@@ -3,21 +3,21 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: CWE-150
```

Comment:
Upvote for this, I think it is interesting how this is something that comes from upstream and affects the system down the line.